src/libs/getSafeAreaPaddingTop/index.android.js

```diff
@@ -0,0 +1,5 @@
+import {StatusBar} from 'react-native';
+
+export default function getSafeAreaPaddingTop() {
+    return StatusBar.currentHeight || 0;
```

Comment:
`StatusBar.currentHeight` value always exists in android so I think `|| 0` is redundant.